src/components/Alert.jsx

```diff
@@ -14,7 +14,7 @@ const Core = function({
 }) {
   return (
     <MuiAlert {...rest}>
-      {children}
+      {children},
```

Comment:
this looks phishy

Reply:
Ah yeah. Vestige of something I was troubleshooting. Weird that the linters were fine with this.